tests/functional/spec/timeout_modal.spec.js

```diff
@@ -0,0 +1,69 @@
+import TimeoutInterstitialPage from "../generated_pages/timeout_modal/timeout-modal-interstitial.page";
+import TimeoutSubmitPage from "../generated_pages/timeout_modal/submit.page";
+import { TimeoutModalPage } from "../base_pages/timeout-modal.page.js";
+
+describe("Timeout", () => {
+  describe("Given I am completing the survey, when the session timeout is set to 5 seconds", () => {
+    before(() => {
+      browser.openQuestionnaire("test_timeout.json");
+    });
+
+    it("It will redirect to signed-out page after session expires", () => {
```

Comment:
We should update all these test to `Given` `When` `Then`.

Reply:
Updated to use `Given`, `When`, `Then`. I used one of the possible styles we already have present in tests, I hope it's the one you wanted.